src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -565,7 +565,7 @@ private static string GetNativeDllSubFolderName(out string ext)
     /// <summary>
     /// This is the managed entry point for Microsoft.PowerShell.CoreCLR.AssemblyLoadContext.dll.
     /// </summary>
-    public class PowerShellAssemblyLoadContextInitializer
+    public static class PowerShellAssemblyLoadContextInitializer
```

Comment:
Breaking change in public API?

Reply:
Yes, this change removes the public constructor for the class so is technically a breaking change. Does creating an instance of `PowerShellAssemblyLoadContextInitializer` have any obvious utility?

Reply:
We need a confirmation from @daxian-dbw

Reply:
reverted

Reply:
Opened #13874